<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FIAS SImulator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../hms/extra_static/favicon.png</normaloff>../../hms/extra_static/favicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="1">
     <widget class="QPushButton" name="sendMessageButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QTextEdit" name="logsTextArea">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="messageInput"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Host</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hostAddressInput">
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hostPortInput">
        <property name="text">
         <string>5010</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnetPushButton">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="eventComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>---------------</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Posting inquery PR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Posting request PR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bill view XR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Remote Checkout XC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Link record LR</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
